### PR TITLE
Html page output

### PR DESCRIPTION
**Pull request by sheth19**
Basic html output of rpm audit

### CODE REVIEW

File: lib/htmlbuilder.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env node
+const fs = require('fs');
+const Handlebars = require('handlebars');
+
+module.exports.buildHtml = (auditData) => {
+    var template = fs.readFileSync('../lib/template.html', 'utf8');
+
+    auditData.forEach(element => {
```
Comment (AScarberry) [inline lib/htmlbuilder.js:8]: Leftover loop? Can probably delete this.

File: lib/htmlbuilder.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        
+    });
+
+    var people = [];
```
Comment (AScarberry) [inline lib/htmlbuilder.js:12]: Rename this array to something more relevant

File: lib/htmlbuilder.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env node
+const fs = require('fs');
+const Handlebars = require('handlebars');
+
+module.exports.buildHtml = (auditData) => {
+    var template = fs.readFileSync('../lib/template.html', 'utf8');
```
Comment (AScarberry) [inline lib/htmlbuilder.js:6]: Try `const template = require('template.html')` above this function declaration under `const Handlebars`

File: lib/template.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+<html>
+<head>
+    <title>Auditly</title>
+</head>
+<body style="background-color:black">
+    <center>
+        <H style="color:#ffec80; font-size:450%;font-family:Verdana;"><strong>Auditly CVE Report<strong></H1>
+        <div id = 'htmlOutput'></div>
+    </center>
+        {{#each people}}
```
Comment (AScarberry) [inline lib/template.html:11]: When you rename your array in your JS, make sure to do it here as well.

File: lib/htmlbuilder.js (hunk 3 above)
Comment (AScarberry) [reply lib/htmlbuilder.js:6]: Never mind, this doesn't.

File: lib/htmlbuilder.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+const Handlebars = require('handlebars');
+const path = require('path');
+
+
+module.exports.buildHtml = (auditData) => {
+    var template = fs.readFileSync(path.join(__dirname, './template.html'), 'utf8');
+
+    var errors = [];
+
+      auditData.forEach(element => {
```
Comment (calebeno) [inline lib/htmlbuilder.js:12]: This a great use case for the Array.map function.
```
var errors = auditData.map((element) => {
...
}
```
Doing this means you don't need to have the `errors.push`.

**Pull request merged**